### ISSUE TITLE
platform type is used as globally set parameter

### ISSUE THREAD

**Issue by rtommy** [telekom/5g-trace-visualizer#32]
The 'platform' is not passed as a parameter thru the functions, it is basically defined globally.

Would not be better to pass it as a parameter for `call_wireshark`?

```
if __name__ == '__main__':
...
    platform = platform.system()
...
    if args.wireshark != 'none':
        input_file = call_wireshark(args.wireshark, input_file, args.http2ports)
```

How about:
```
input_file = call_wireshark(args.wireshark, **platform**, input_file, args.http2ports)
...
output_file = call_wireshark_for_one_version(wireshark_version, **platform**, input_file_str, http2ports_string, mode, check_if_exists)
```

Since in `call_wireshark_for_one_version` the `platform` is used few times and if the `wireshark_version` is properly received as an input parameter, the `platform` should be too.

E.g.:
```
        if wireshark_version == 'OS' and (platform == 'Linux'):
            mergecap_path  = "/usr/bin/mergecap"
        elif wireshark_version == 'OS' and (platform != 'Linux'):
            mergecap_path = 'mergecap'
```

**Comment by jkolom** [comment]
Hi!
Well, I would say that since it works, I would leave it like this. But feel free to do a pull request. It is true that it would be better practice, but it is not per se wrong 😉 
BR,
Josep

**Comment by rtommy** [comment]
Hi Josep,

The pull request has been there since last month: [https://github.com/telekom/5g-trace-visualizer/pull/33](url)

Yes, the code works but overall it seems you always used parameters for each function to pass variables to it and not globally referring to any. That is why I thought I would mention it. It is just a minor code enhancement.

Cheers!

**Comment by jkolom** [comment]
Hi!
Sorry, I missed that one. Normally I would an email from github. Probably it wrongly ended in Spam 😅 
I just merged your pull request in 4575631.
Thanks for the contribution! 😄
